roles/splunk_common/tasks/disable_popups.yml

```diff
@@ -0,0 +1,15 @@
+---
+  - name: Disable Popups
+    uri:
+      url: "{{ cert_prefix }}://127.0.0.1:{{ splunk.svc_port }}/{{ item.key }}"
+      method: POST
+      user: "{{ splunk.admin_user }}"
+      password: "{{ splunk.password }}"
+      body: "{{ item.value }}"
+      validate_certs: false
+      status_code: 200,201,409
+    with_items:
+      - { key: "servicesNS/admin/user-prefs/data/user-prefs/general", value: "hideInstrumentationOptInModal=1&notification_python_3_impact=false&showWhatsNew=0&dismissedInstrumentationOptInVersion=4" }
+      - { key: "servicesNS/nobody/splunk_instrumentation/admin/telemetry/general", value: "showOptInModal=0&optInVersionAcknowledged=4" }
```

Comment:
I would be careful about hardcoding this `optInVersionAcknowledged=4` here? Seems like this is being compared with `optInVersion` and that may be an unbounded number, so a future release may render this change ineffective.

https://docs.splunk.com/Documentation/Splunk/8.0.5/Admin/Telemetryconf